app/components/feedback-button/template.hbs

```diff
@@ -1,46 +1,50 @@
-{{#bs-modal class=modalClass open=open onSubmit=(action "submit") onHidden=(action 'hideDialog') as |modal|}}
-    {{#modal.body}}
-        {{#if active}}
-            <div class="text-center">
-                <h2 class="Modal__title">{{ t 'feedback.title' }}</h2>
-                <h4 class="Modal__text">{{text}}</h4>
-            </div>
-            <div class="form-group">
-                {{textarea class="form-control" rows=dialogRows autofocus="true" placeholder=(t 'feedback.placeholder') value=body}}
-            </div>
-            <div class="checkbox">
-                <label>
-                    {{input type="checkbox" checked=followup}}
-                    {{t 'feedback.follow_up_label'}}
-                </label>
-            </div>
-        {{/if}}
-        {{#if success}}
-            {{success-icon}}
-            <div class="text-center">
-                <h2>{{t 'feedback.thank_you'}}</h2>
-                <h3>{{t 'feedback.success'}}</h3>
-            </div>
-        {{/if}}
-    {{/modal.body}}
-    {{#modal.footer}}
-        {{#if active}}
-            {{#bs-button onClick=(action modal.close)}}
-                {{t 'general.cancel'}}
-            {{/bs-button}}
-            {{#bs-button type="success" onClick=(action modal.submit)}}
-                {{t 'feedback.confirm_button_text'}}
-            {{/bs-button}}
-        {{/if}}
-        {{#if success}}
-            {{#bs-button type="success" onClick=(action modal.close)}}
-                {{t 'feedback.dismiss'}}
-            {{/bs-button}}
-        {{/if}}
-    {{/modal.footer}}
-{{/bs-modal}}
+{{#if enabled}}
+    {{#bs-modal class=modalClass open=open onSubmit=(action "submit") onHidden=(action 'hideDialog') as |modal|}}
```

Comment:
This works, but conditionally adding the hidden to the button would also be just as effective. The modal would never be displayed, because it's not clickable.